README.md

```diff
@@ -210,62 +216,148 @@ The basic properties that must be configured for the connector are:
-| client.proxy.pref.auth.type       | The authentication type to use with the client proxy. Must be one of `BASIC`, `DIGEST` or `X_ABLY_TOKEN`. Requires `client.proxy` to be set to `true`.                                                                          | *String*  | Basic   |
-| client.push.full.wait             | Sets whether Ably should wait for all the effects of push REST requests before responding.                                                                                                                                      | *Boolean* ||
-| client.tls                        | Sets whether TLS is used for all connection types.                                                                                                                                                                              | *Boolean* | True    |
-| client.token.params               | Sets whether the configured token parameters are used.                                                                                                                                                                          | *Boolean* ||
-| client.token.params.capability    | Stringified JSON capability requirements for the token. When omitted, the REST API default to allow all operations is applied by Ably, with the string value `{“*”:[“*”]}`. Requires `client.token.params` to be set to `true`. | *String*  ||
-| client.token.params.client.id     | The client ID to include with the token. Requires `client.token.params` to be set to `true`.                                                                                                                                    | *String*  ||
-| client.token.params.ttl           | The requested time to live (TTL) for the token in milliseconds. When omitted, the REST API default of 60 minutes is applied by Ably. Requires `client.token.params` to be set to `true`.                                        | *Boolean* | 0       |
-| client.loglevel                   | Sets the verbosity of logging.                                                                                                                                                                                                  | *Integer* | 0       |
+| Property                          | Description                                                                                                                                            | Type      | Default |
+|-----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|---------|
+| message.name                      | Ably message name to publish. Can be a pattern, as per `channel` above.                                                                                | *String*  |         |
```

Comment:
I believe the context for patterns is in `Dynamic Channel Configuration`, with channel just directing to that section, so we could just direct to that section instead.

Reply:
Done in https://github.com/ably/kafka-connect-ably/pull/142/commits/aba1da6aab4ddd28ec80d4ef51080c3195a73a26